proto/pdpb.proto

```diff
@@ -250,7 +250,11 @@ message BootstrappedError {
     
 }
 
+message StoreIsTombstoneError {
+}
+
 message Error {
-    optional string message                 = 1;
-    optional BootstrappedError bootstrapped = 2;
+    optional string message                     = 1;
+    optional BootstrappedError     bootstrapped = 2;
```

Comment:
I think is fine to add `Error` suffix to error message, just like request message named `IsBootstrappedRequest`.